README.md

```diff
@@ -0,0 +1,52 @@
+
+# Flick-fil-a
+
+This is a movie search web application that allows you to search any movie within the database. This app also allows for adding movies to your favorites and storing them for later recall.
+
+
+## Run Locally
+
+Clone the project
+
+```bash
+  git clone https://link-to-project
+```
+
+Go to the project directory
+
+```bash
+  cd my-project
+```
+
+Install dependencies
+
+```bash
+  npm install
+```
+
+Start the server
+
+```bash
+  npm run start
+```
+
+
+## Deployment
+
+To deploy this project run
+
+```bash
+  npm run build
+```
+
+
+## Tech Stack
+
+**Client:** TypeScript
+
+**Server:** Node, Express
+
+
+## Support
+
+For support, email tyseawood@protonmail.com.
```

Comment:
it feels risky to leave an email in public (bots might spam you) but I've had my gmail in my profile forever and haven't noticed any issues. I guess whatever that is worth ha

Reply:
I agree with you, I can put a info@gmail.com or something here instead.